src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {
+  Nav,
+  NavLink,
+  Bars,
+  NavMenu,
+  NavBtn,
+  NavBtnLink,
+} from './NavbarElements.jsx';
+
+export default function Navbar() {
+  return (
+    <>
```

Comment:
same as per this comment - https://github.com/chevalier88/capstone-project-management-app-frontend/pull/1/commits/84df193b474535ad40b1cb872f660a2c96cadf58#r908405355